src/Storage/Storage.Management/help/Copy-AzStorageBlob.md

```diff
@@ -37,7 +37,7 @@ Copy-AzStorageBlob -AbsoluteUri <String> -DestContainer <String> -DestBlob <Stri
 ```
 
 ## DESCRIPTION
-The **Copy-AzStorageBlob** cmdlet copies a blob synchronously, currently only support block blob.
+The **Copy-AzStorageBlob** cmdlet copies a blob synchronously, currently only support block blob. This will create any missing parent folders in the DestBlob parameter.
```

Comment:
This is not correct.
For Blob, the Directory are virtual directory (not really exist). So Powershell actually won't create the parent directory, but will just copy the destination blob.